src/Tank.hpp

```diff
@@ -28,36 +29,42 @@ class TankPart {
   Rotation mRotation = Rotation::None;
 };
 
+struct TankTextures {
+  std::reference_wrapper<sf::Texture> mBody;
+  std::reference_wrapper<sf::Texture> mTower;
+  std::reference_wrapper<sf::Texture> mShot;
+  std::reference_wrapper<sf::Texture> mTracks;
```

Comment:
If I understand this correctly - reference wrapper is only to have nice initialization, right?

Reply:
I used reference wrapper to make `TankTextures` copyable. The point of `TankTextures` structure is to lower the amount of arguments required by `Tank` and to avoid possible mistakes by passing arguments in the wrong order (we have 4 textures next to each other).